aem65/src/main/java/com/cognifide/qa/bb/aem65/tests/pageobjects/corecomponents/teaser/TeaserComponentImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.cognifide.qa.bb.aem65.tests.pageobjects.corecomponents.teaser;
+
+import com.cognifide.qa.bb.constants.HtmlTags;
+import com.cognifide.qa.bb.qualifier.CurrentScope;
+import com.cognifide.qa.bb.qualifier.PageObject;
+import com.cognifide.qa.bb.wait.BobcatWait;
+import com.google.inject.Inject;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.springframework.util.Assert;
+
+import java.util.List;
+
+@PageObject (css = ".cmp-teaser")
+public class TeaserComponentImpl implements TeaserComponent {
+
+    @Inject
+    @CurrentScope
+    private WebElement component;
+
+    @FindBy (css = ".cmp-image__image")
+    private WebElement image;
+
+    @FindBy (css = ".cmp-teaser__title")
+    private WebElement title;
+
+    @FindBy (css = ".cmp-teaser__title-link")
+    private WebElement titleLink;
+
+    @FindBy (css = ".cmp-teaser__description > p")
+    private WebElement description;
+
+    @FindBy (css = ".cmp-teaser__description")
+    private List<WebElement> elements;
+
+    @FindBy (css = ".cmp-teaser__description")
+    private WebElement isDescriptionVisible;
+
+    @Inject
+    private BobcatWait bobcatWait;
+
+    public void checkDescriptionVisibility(){
+        Assert.isTrue(elements.isEmpty(),"Element exist when it shouldn't");

```

Comment:
Please keep assertions in test classes only, for maximum visibility and separation

Reply:
fixed, I moved assertion into test class